docs/dev/news/4205.fix

```diff
@@ -0,0 +1 @@
+Handling of opaque methods in traits overriding check
```

Comment:
It would be nice to have a longer, more explanatory message here.

Reply:
Maybe "Traits with opaque functions can now be extended without errors" or something like that.